.github/workflows/package-smoke-test.yaml

```
@@ -45,23 +45,26 @@ jobs:
            uses: oven-sh/setup-bun@v2
 
          - name: Install dependencies
-           run: bun install
+            run: bun install
```

actionlint (1.7.9)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)